### DISCUSSION TITLE
Box64 on rpi3b+

### DISCUSSION THREAD

**Discussion by DotJARgon** [ptitSeb/box64#245] [Q&A]
What modifications would be necessary for box64 to run on the rpi3b+? I see that this is supported on the pi4 with a 64 bit operating system, but the rpi3b+ from what I understand too can run a 64 bit os, is there anything special about installing on an rpi3b+?

**Comment by ptitSeb** [comment]
There is nothing special needed. Just build with generic arm configuration and you should have a working build.

**Reply by DotJARgon** [reply]
Thank you, I forgot to respond but it did end up working. I have been trying to install steam, which it did, and after the installation it refuses to open beyond the extracting packages screen. I am getting errors that symlink 15 failed, failure to unzip packages, as well as finally the error is cannot open steamui.so. is there any way around these problems?

**Reply by ptitSeb** [reply]
That sound like a storage issue. Did you have enough space on your sdcard? And what type of partition format is this? Ext2/Ext4/file2fs/other?
I never had any issue like this on my side on any platform I installed steam.

**Reply by ptitSeb** [reply]
Also, make sure your box86 is up to date.

**Reply by DotJARgon** [reply]
So I just double checked, I just used the normal rpi installer for the 32 bit platform since 64 wouldn't go anywhere, my sd card is a 64 gb one, with a 2gb swap space (since i kept running into issues with that) and box86 is up to date using the instructions on your GitHub, I even did a git pull and make again and nothing changed

**Reply by ptitSeb** [reply]
and what `df -h` gives you?

**Reply by DotJARgon** [reply]
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        58G  6.8G   48G  13% /
devtmpfs        333M     0  333M   0% /dev
tmpfs           462M     0  462M   0% /dev/shm
tmpfs           185M  840K  184M   1% /run
tmpfs           5.0M  4.0K  5.0M   1% /run/lock
/dev/mmcblk0p1  253M   49M  204M  20% /boot
tmpfs            93M   20K   93M   1% /run/user/1000

This is what it gives me, sorry for the late response I was working all day.

**Comment by ptitSeb** [comment]
There is no reason for steam installation to fail, unless there is some physical issue with the sdcard. You should remove all traces of steam (look in `~/.local/share/Steam` ) and try again with latest version of box86. If you have error, check with `dmesg` (or `sudo dmesg`) if you see some error accessing the sdcard.

**Reply by DotJARgon** [reply]
I removed steam, then installed it again, but before doing so I added i386 and it worked, not sure if that was the problem or not, but hopefully that might help anyone who looks at this
